packages/asc-ui/src/components/ContextMenu/ContextMenu.tsx

```diff
@@ -36,8 +37,7 @@ const ContextMenu: React.FC<
     React.Children.toArray(children).forEach((child) => {
       // @ts-ignore
       if (child && child.type !== ContextMenuItem) {
-        // eslint-disable-next-line no-console
-        console.warn(
+        deprecatedWarning(
           'Warning: you are rendering a different component type in <ContextMenu /> other than <ContextMenuItem />.',
```

Comment:
I think we can drop the `Warning:` part of this message, as we're already using `console.warn` which should give enough formatting.

Reply:
Also, this warning is not about something that's deprecated...